### PR TITLE
Feat: update mobile view restriction

### PR DESCRIPTION
**Pull request by joshuanatanielnm**
#576 

## Description

Remove mobile restriction on home page, update cellars order, and update mobile restriction text

## Changes

List any technical changes.

- [x] Remove mobile restriction on home page
- [x] update cellars order on cellar data map
- [x] update mobile restriction text 

## Screenshots (if appropriate):
image
image
image

### CODE REVIEW

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -16,21 +16,23 @@ export const Layout: VFC<FlexProps> = ({ children, ...rest }) => {
   const { chain } = useNetwork()
 
   const router = useRouter()
-  const isStrategiesLandingPage =
-    router.pathname.split("/")[1]?.toLowerCase() === "strategies" &&
-    router.pathname.split("/")[3]?.toLowerCase() !== "manage"
+
+  const isHomeOrStrategiesLandingPage =
+    router.pathname === "/" ||
+    (router.pathname.split("/")[1]?.toLowerCase() === "strategies" &&
```
Comment (xanderjl) [inline src/components/Layout.tsx:22]: For these last two lines, could we not use something like `router.pathname.includes("strategies")`, or do you feel that wouldn't be specific enough? [Example docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

**Review by xanderjl — CHANGES_REQUESTED** [review]
One potential change to improve readability, but may not work the way I think it will.

File: src/components/Layout.tsx (hunk 1 above)
Comment (xanderjl) [reply src/components/Layout.tsx:22]: `Array.includes` may also cover future cases where routing patterns change for whatever reason 👀

Comment (joshuanatanielnm) [reply src/components/Layout.tsx:22]: Thanks Xan, I think this implementation is much better 👍

**Review by xanderjl — APPROVED** [review]
LGTM! 💪

**Pull request merged**